### PR TITLE
Hopeful 0.92 release (Merge of distributed_rewrite/config_work)

### PR DESCRIPTION
**Pull request by drallensmith**
socket.getfqdn is unfortunately (as stated in socket) rather dependent on the local socket/inet/etc configuration to work as expected. (It also prefers names that have "."s in them, incidentally.) I've added what it produces for my machine (OS X 10.12) for "localhost" and "127.0.0.1" to the recognized synonyms for the loopback address. I have also put in a modification to the test_distributed.py code that makes it more informative if an error happens.

This problem is understandable; I would not expect @bennr01 to have around every variety of machine. It should be possible to get Travis to test on more than one OS, though - the LARG/HFO project does that; I'll add an updated .travis.yml to at least also check OS X.

### CODE REVIEW

**Comment by drallensmith** [comment]
It may be worthwhile to look into the "trusty" method for a speedup on Travis, BTW. Admittedly, it can be even trickier than multiple OSes on what's available, from what I'm reading.

**Comment by drallensmith** [comment]
Sigh. Looks like the trusty build, with or without sudo, will be needed. I'll reset back to the first version of the PR and try to take a look at that.

**Comment by drallensmith** [comment]
From the trusty build, some setups w/virtual machines or similar aren't going to recognize "::1", unfortunately.

**Comment by drallensmith** [comment]
It may be possible to get .travis.yml working with OS X by not specifying the version of python with that, since the base system does include python and allow pip installs. However, the question will be how to do that without cutting down on multiple-version checking on Linux.

File: .coveralls.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-service_name: travis-pro
-repo_token: eoFbGKxivT1jjyPsUJOuzvrTDDrtCLQQw
```
Comment (drallensmith) [inline .coveralls.yml]: Odd; I put .coveralls.yml into .gitignore so it wouldn't overwrite the master branch copy! Sorry about that...

**Comment by CodeReclaimers** [comment]
Wow, love the huge improvements to the docs on top of everything else! Is this at a point where you're good with releasing it?

**Comment by drallensmith** [comment]
Quite welcome; I enjoy helping! Yes, I'd say I'm comfortable with it as a release.

**Comment by drallensmith** [comment]
Conflict re .coveralls.yml fixed; docstring mention in distributed.py of beta status.

**Pull request merged**